src/Sylius/Bundle/CoreBundle/spec/StateMachineCallback/OrderItemUnitCallbackSpec.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace spec\Sylius\Bundle\CoreBundle\StateMachineCallback;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use SM\Factory\FactoryInterface;
+use SM\StateMachine\StateMachineInterface;
+use Sylius\Bundle\CoreBundle\StateMachineCallback\OrderItemUnitCallback;
+use Sylius\Component\Core\Model\OrderItemUnitInterface;
+use Sylius\Component\Core\Model\ShipmentInterface;
+use Sylius\Component\Shipping\ShipmentTransitions;
+
+/**
+ * @author Robin Jansen <robinjansen51@gmail.com>
+ */
+class OrderItemUnitCallbackSpec extends ObjectBehavior
+{
+    function let(FactoryInterface $factory) {
+        $this->beConstructedWith($factory);
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType(OrderItemUnitCallback::class);
+    }
+
+    function it_updates_shipment_state_when_no_unit_in_backorder(
+        $factory,
+        OrderItemUnitInterface $unit,
+        OrderItemUnitInterface $unit1,
```

Comment:
I would rename this two variables into ``$unit1/$unit2`` or ``$firstUnit/$secondUnit``.